infotv/frontend/package.json

```diff
@@ -4,49 +4,46 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "build": "webpack --verbose --progress",
-        "watch": "webpack --verbose --progress --watch",
-        "release": "webpack -p",
-        "lint": "eslint src webpack.config.js --ext .js,.jsx",
+        "build": "webpack --progress",
+        "watch": "webpack --mode=development --progress --watch",
+        "release": "webpack --mode=production",
+        "lint:ts": "tslint src/**.ts*",
```

Comment:
These days you should use `eslint` instead of `tslint` for TypeScript linting. I just pasted an example config to #desutech couple of days ago 😄